### PR TITLE
COMMAND: Add ability to run a mission autonomously

### PR DESCRIPTION
**Pull request by mattlangford**
Theres some various sim stuff and I fixed the annoying import errors we were getting too.

I can't reliably spell 'autonomously', so I used 'auto' everywhere instead

### CODE REVIEW

File: command/sub8_alarm/listeners/network_listener.py [hunk 1]
```diff
@@ -1,34 +1,59 @@
...
+        
+        # Make sure we don't accidentally let the sub loose.
+        # We need auto_msgs_req messages before we go autonomous mode.
+        self.auto_msgs_req = autonomous_msgs_req
+        self.auto_msg_count = 0
+
+        self.sub = rospy.Subscriber('/keep_alive', String, self.got_network_msg, queue_size=1)
+        self.auto_service = rospy.ServiceProxy('/go_auto', Empty)
         self.alarm_broadcaster, self.alarm = single_alarm('network-timeout', severity=1)
         rospy.Timer(rospy.Duration(0.01), self.check)
```
Comment (rleyva) [inline command/sub8_alarm/listeners/network_listener.py]: Should we really be calling back 100 times a second?

**Comment by rleyva** [comment]
At this moment it does not look like we will be using the gripper, so no need to add it for now.

File: command/sub8_alarm/listeners/network_listener.py (hunk 1 above)
Comment (mattlangford) [inline command/sub8_alarm/listeners/network_listener.py]: We have like 16 cores now, there are no rules. I'll dial it back in the morning

Comment (rleyva) [inline command/sub8_alarm/listeners/network_listener.py]: ~0.25 sec be okay for this

**Pull request merged**